src/core/Config.h

```diff
@@ -238,11 +238,12 @@ class Config : public QObject
     static QPair<QString, QString> defaultConfigFiles();
 
     static QPointer<Config> m_instance;
+    static bool m_isPortable = QFile::exists(QCoreApplication::applicationDirPath().append("/%1").arg(".portable"));
```

Comment:
No check if the file is writable, as the file is never written to

Reply:
Don't remove code checks. I think we'll just take this from here thank you

Reply:
the check was previously whether the keepassxc.ini is writable as part of the portable-check. As the .portable file is never written to, it didn't seem correct to check it.
The original check was removed here: https://github.com/keepassxreboot/keepassxc/pull/10760/commits/d392402104fdf556e89a3fcd9f87c0234a75326d#diff-0cd3d444e91ef419d6405be18ac17150261864f39dfe5f6b6018680a67aa77afL522

Reply:
It was checked to make sure the exact problem that started this conversation doesn't happen...

Reply:
well yes, but that check didn't work, as it still passed, despite not being able to write to that location.